### PR TITLE
[LAB 1] Added tests for starting script, fixed reference for tokenizer

### PR DESCRIPTION
**Pull request by dmitry-uraev**

### CODE REVIEW

File: config/start_py_test.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import argparse
+import re
+import sys
+
+
+def check_assert_is_in_file(content: str):
+    print(content)
+    expected = 'assert RESULT'
+    actual = re.findall(expected, content)
+    if expected == actual or expected in content:
```
Comment (WhiteJaeger) [inline config/start_py_test.py]: `expected` is a string and `actual` is a list, so they cannot be equal.

File: config/start_py_test.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    return 1
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Checks start.py files and tests them')
+    parser.add_argument('--start_py_content', type=str, help='Content of start.py for each lab')
+    args: argparse.Namespace = parser.parse_args()
+
+    result = check_assert_is_in_file(args.start_py_content)
+    if result == 1:
```
Comment (WhiteJaeger) [inline config/start_py_test.py]: result is either 0 or 1, so better would be:
```suggestion
    if result:
```

File: config/start_py_test.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# pylint: skip-file
```
Comment (WhiteJaeger) [inline config/start_py_test.py]: I do not really understand this test. It checks for the line that is already in `start.py` file, so this test will pass - why do we need it then?

Comment (demid5111) [inline config/start_py_test.py]: do not skip this file from lint

File: config/start_py_test.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+import argparse
```
Comment (demid5111) [inline config/start_py_test.py]: rename to reflect content - it is not a test, it is a check script, check_start_content

File: config/start_py_test.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+import re
+import sys
+
+
+def check_assert_is_in_file(content: str):
```
Comment (demid5111) [inline config/start_py_test.py]: is_assert_in_file() -> bool

File: config/start_py_test.py (hunk 1 above)
Comment (demid5111) [inline config/start_py_test.py]: return condition directly

File: config/run-start.sh [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+echo "Running start.py checks..."
+
+LABS=$(cat config/labs.txt)
+WAS_FAILED=0
+
+for lab in $LABS; do
+	echo "Running start.py checks for lab #${lab}"
+
+	if ! python3 lab_"${lab}"/start.py;  then
+    	WAS_FAILED=1
```
Comment (demid5111) [inline config/run-start.sh:11]: formatting

**Review by demid5111 — CHANGES_REQUESTED** [review]

File: config/check_start_content.py [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+"""
+
+import argparse
+import sys
+
+
+def check_assert_is_in_file(content: str):
+    expected = 'assert RESULT'
+    if expected in content:
+        return True
```
Comment (WhiteJaeger) [inline config/check_start_content.py]: You do need need extra line, just return the result of the condition.

File: config/check_start_content.py [hunk 8]
```diff
@@ -0,0 +1,28 @@
+"""
+Some simple checks for start.py lab files
+"""
+
+import argparse
+import sys
+
+
+def check_assert_is_in_file(content: str):
```
Comment (WhiteJaeger) [inline config/check_start_content.py]: Please, specify return type and use shorter name.

**Review by WhiteJaeger — CHANGES_REQUESTED** [review]

**Pull request merged**